### ISSUE TITLE
Incorrect constant export example

### ISSUE THREAD

**Issue by takrishna** [airbnb/javascript#3032]
> _What about all const variables? - This is unnecessary, so uppercasing should not be used for constants within a file. **It should be used for exported constants however.**_

There are two points:

1. Based on the above description the provided example appears incorrect
2. Also, Since const variables can't be changed through reassignment, this example (i.e., variable name) feels odd. 

Current example:
```
// bad
export const THING_TO_BE_CHANGED = 'should obviously not be uppercased';
```

Perhaps it could be:
```
// bad
export const some_exported_constant = 'should obviously be uppercased';
```

**Comment by ljharb** [comment]
"constants" in this context is different than a `const` variable.

**Comment by takrishna** [comment]
If we read the highlighted (bold) in the quoted section above, it basically means that exported constants should to be capitalized, but then the example is right opposite (it says _"should obviously not be uppercased"_.)

With respect to the second point around const, as per ECMASCRIPT specification it is NOT possible to change/reassign an exported const variable, but then the variable naming (in this example i.e., const THING_TO_BE_CHANGED) suggests that it is possible :)

**Comment by ljharb** [comment]
because a string with multiple words in it isn't a conceptual constant.

**Comment by takrishna** [comment]
> because a string with multiple words in it isn't a conceptual constant.

irrespective of being a "conceptual" constant, it would not be possible to mutate/modify a variable declared using the "const" keyword that is assigned with a primitive as its value (in this case a "string"), hence the variable naming "THING_TO_BE_CHANGED" is not justified. It can never be changed, the JS spec simply doesn't allow it.

Here the rule is about "const" variables
> _What about all_ **const** _variables?_

If it was about "constant" variables (including say "conceptual constants") I would have hoped it to read as:
>_What about all_ **constant** _variables?_

In my opinion the rule is correct but only the example is misplaced. The rule clearly states that an exported "const" variable should be capitalized and a "const" variable within a file need NOT be capitalized. The example I am referring in this issue is "**exporting**" a const variable and hence should be capitalized.

In my humble view, constants could very well be made of multiple words, e.g., official country names are perfect examples:
`const NETHERLANDS = 'The Netherlands';`
`const USA = 'United States of America;`

**Comment by ljharb** [comment]
True, that would make more sense.

**Comment by Shlokmonster** [comment]
Hey! I’d love to work on this issue. Can you assign it to me?

**Comment by ljharb** [comment]
@Shlokmonster no, nothing in open source is assigned to people - you just make a PR if you want to work on it. However, note #3072 already exists, so if you have changes to that, instead of opening a PR, please comment on it with a link to your branch.
